### PR TITLE
Added method to return active span

### PR DESCRIPTION
**Pull request by ujwaldhakal**
Added a method to return the active span so that one can have flexibility to hook other process.

The current Symfony bundle is not compatible with Symfony Messenger. So I had to hook with these [messenger events](https://symfony.com/doc/current/messenger.html#messenger-events) for this I need tracer object to manually close & flush if there are any active connections

### CODE REVIEW

File: Service/TracingService.php [hunk 1]
```diff
@@ -36,6 +36,11 @@ public function injectTracingHeadersIntoCarrier(array $carrier): array
         return $this->doInjectTracingHeadersIntoCarrier($span, $carrier);
     }
 
+    public function getActiveSpan(): ?Span
+    {
+        return $this->tracer->getActiveSpan()
```
Comment (cawolf) [inline Service/TracingService.php:41]: The pipeline reported whitespaces at the end of the line. I think we are also missing a `;` here.
```suggestion
        return $this->tracer->getActiveSpan();
```

**Review by cawolf — CHANGES_REQUESTED** [review]
Thank you for your contribution. It would be awesome, if you could add a unit test for the new functionality as well.

File: Service/TracingService.php (hunk 1 above)
Comment (ujwaldhakal) [reply Service/TracingService.php:41]: Sure, I will update them and will try to write an example on how it works with Symfony Messenger

**Comment by ujwaldhakal** [comment]
> Thank you for your contribution. It would be awesome, if you could add a unit test for the new functionality as well.

Sure, I will have look them tomorrow

**Pull request closed without merging**